pxtlib/initprj.ts

```diff
@@ -54,7 +54,7 @@ ${lf("To edit this repository in MakeCode.")}
 ${lf("This image shows the blocks code from the last commit in master.")}
 ${lf("This image may take a few minutes to refresh.")}
 
-![${lf("A rendered view of the blocks")}](https://github.com/@REPO@/raw/master/.makecode/blocks.png)
+![${lf("A rendered view of the blocks")}](https://github.com/@REPO@/raw/master/.github/makecode/blocks.png)
```

Comment:
This will break the links for any existing project readmes, right? as the readme will remain the old location but the files will be stored in the new location